graylog2-web-interface/src/components/configurations/DecoratorsConfig.tsx

```diff
@@ -37,7 +38,7 @@ const { DecoratorsActions } = CombinedProvider.get('Decorators');
 const DecoratorsConfig = () => {
   const [streams, setStreams] = useState<Array<Stream> | undefined>();
   const [currentStream, setCurrentStream] = useState(DEFAULT_STREAM_ID);
-  const [decorators, setDecorators] = useState();
+  const [decorators, setDecorators] = useState<Array<Decorator>>();
```

Comment:
In similar cases we type this as `Array<Decorator> | undefined`, but we can adjust this case when we enable `strictNullChecks`

Reply:
Good catch. I prefer to change it right now!

Reply:
:white_check_mark:

Reply:
:white_check_mark: